types/rdfjs__fetch-lite/Factory.d.ts

```diff
@@ -0,0 +1,20 @@
+import { BaseQuad, DatasetCore, Quad, Stream } from '@rdfjs/types';
+import { FormatsInit } from '.';
+
+interface RdfFetchResponse<D extends DatasetCore<OutQuad, InQuad>, OutQuad extends BaseQuad = Quad, InQuad extends BaseQuad = OutQuad> extends Response {
+    quadStream(): Promise<Stream<OutQuad>>;
+    dataset(): Promise<D>;
```

Comment:
What about the config function (https://github.com/rdfjs-base/fetch-lite/blob/425f5083e7df0a97b7fb5d82c0c616ae5aac5969/Factory.js#L15-L17)?